pkg/api/login.go

```diff
@@ -35,8 +35,10 @@ func (hs *HTTPServer) validateRedirectTo(redirectTo string) error {
 	if to.IsAbs() {
 		return login.ErrAbsoluteRedirectTo
 	}
-	if hs.Cfg.AppSubUrl != "" && !strings.HasPrefix(to.Path, "/"+hs.Cfg.AppSubUrl) {
-		return login.ErrInvalidRedirectTo
+	if hs.Cfg.AppSubUrl != "" {
```

Comment:
Is there a reason for dividing it into two if statements, instead of and-ing the conditions as before? Personally I'd prefer keeping the one if statement, since it makes it clear that there's no possibility of further branches nested within.

Reply:
this can be combined back into one statement, that was just an artifact of debugging